### PR TITLE
add cli for universal-dotenv

### PR DESCRIPTION
**Pull request by fastner**
Add CLI interface

### CODE REVIEW

**Comment by fastner** [comment]
add support for -c <context> in unienv binary

**Comment by swernerx** [comment]
The other PR was merged.

Do we need any Babel Config adjustments to correctly allow for the TS code added?

**Comment by fastner** [comment]
@swernerx No, I don't think we need babel config changes here. At least "npm pack" output looks good.

File: package.json [hunk 1]
```diff
@@ -54,23 +56,29 @@
...
     "@babel/core": "^7.5.5",
     "@effective/eslint-config": "^0.7.12",
+    "@types/yargs": "^13.0.0",
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^24.8.0",
     "babel-preset-edge": "^5.3.0",
     "eslint": "^6.1.0",
+    "eslint-plugin-jsdoc": "^15.7.2",
+    "eslint-plugin-node": "^9.1.0",
     "husky": "^3.0.1",
```
Comment (swernerx) [inline package.json]: These should not be needed.

File: readme.md [hunk 2]
```diff
@@ -61,9 +61,7 @@ Note: `local` files without `NODE_ENV` are not respected when running in `NODE_E
 All loading features are enabled by importing the core module itself:
 
 ```js
-import { init } from "universal-dotenv"
-
-init()
+import "universal-dotenv"
 ```
```
Comment (swernerx) [inline readme.md:65]: Thanks for this!

**Review by swernerx — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (fastner) [reply package.json]: Fixed

**Pull request merged**